esmvaltool/config-references.yml

```
@@ -656,6 +656,7 @@ references:
   massonnet12tc: "Massonnet, F. et al., The Cryosphere, doi: 10.5194/tc-6-1383-2012, 2012."
   mehran14jgr: "Mehran, A. et al., J. Geophys. Res., doi: 10.1002/2013JD021152, 2014."
   manubens18ems: "Manubens, N., et al., ENVIRON MODELL SOFTW, doi:10.1016/j.envsoft.2018.01.018, 2018."
+  massonet18ncc: "Massonnet et al. Nat. Clim. Change, 8(7), 599–603. https://doi.org/10.1038/s41558-018-0204-z, 2018."
```

```suggestion
  massonet18natcc: "Massonnet et al., Nat. Clim. Change, doi:10.1038/s41558-018-0204-z, 2018."
```
and please provide a bibtex for this reference.